### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jliberma**
I want to format the homepage just like deep learning examples, this will require us to restructure the directories a bit. in a nutshell, we want directories for each framework and then a tools directory. then we have a table on the readme so people can navigate the examples quickly. i need help figuring out what meta data to track in the table for the llm examples. these will be linked out to nemotron on HF most likely, and open source models.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
+|------------------------------------------------------------------------------------------------------------------------|-------------|------|-----------|------------|----------|------|-----------------------------------------------------------------------------------------------------------|------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Langchain  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Llama Index  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
```
Comment (sumitkbh) [inline README.md:24]: We have a single RAG example. We should have single row in this table with framework set as Langchain+Llama Index.
Embedding should be e5-large-v2.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
+- Release notes for each of the NVIDIA optimized containers
+- Links to developer documentation
 
-## What are NVIDIA AI Workflows?
------------------------------
-NVIDIA AI Workflows are intended to provide reference solutions of how to leverage NVIDIA frameworks to build AI solutions for solving common use cases. These workflows provide guidance like fine tuning and AI model creation to build upon NVIDIA frameworks. The pipelines to create applications are highlighted, as well as opinions on how to deploy customized applications and integrate them with various components typically found in enterprise environments, such as components for orchestration and management, storage, security, networking, etc.
+## Retrieval Augmented Generation (RAG)
 
-By leveraging an AI workflow for your specific use case, you can streamline development of AI solutions following the example provided by the workflow to:
+A RAG pipeline embeds multimodal data --  such as documents, images, and video -- into a database connected to a Large Language Model.  RAG lets users to chat with their own data.
```
Comment (sumitkbh) [inline README.md:19]: typo: RAG lets users chat

File: README.md [hunk 3]
```diff
@@ -1,39 +1,40 @@
...
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Llama Index  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
+ 
 
-Using the example workflow provided in this repository, you know exactly what AI framework to use, how to bring data into the pipeline, and what to do with the data output. AI Workflows are designed as microservices, which means they can be deployed on Kubernetes alone or with other microservices to create a production-ready application for seamless scaling. The workflow cloud deployable package can be used across different cloud instances and is automatable and interoperable.
+## Large Language Models
+The NVIDIA family of Large Language Models (LLMs) is optimized for building production-ready generative AI applications for the enterprise.
```
Comment (sumitkbh) [inline README.md:28]: Llama2 is OSS LLM.

Comment (jliberma) [reply README.md:28]: I am going to add links to the Nemotron models on HF in this table.

File: README.md (hunk 1 above)
Comment (jliberma) [reply README.md:24]: okay updated

**Review by sumitkbh — APPROVED** [review]

**Pull request merged**